plugin.json

```diff
@@ -49,9 +49,10 @@
         {
             "key": "deduplicateEvents",
             "name": "Attempt deduplicating events",
```

Comment:
Given we have yes and no, might make sense wording as a question?

Reply:
You mean something like: "Attempt deduplicating events?"

Reply:
Yeah, for example